.github/workflows/cd.yml

```diff
@@ -134,14 +132,12 @@ jobs:
               git push -d origin $TEMPLATE_VERSION
           fi
 
-      - name: generate templates v3
+      - name: rebuild templates after versioning
         run: |
-          .github/scripts/template-zip-autogen-v3.sh $TEMPLATE_PATH
-          cd ./packages/fx-core
-          rm -rf ./templates/fallback
-          node ./scripts/download-templates-zip.js
+          cd ./templates
+          npm run build
```

Comment:
why need rebuild?

Reply:
Templates' package.json may be changed after versioning, which may sync up sdk version.

Reply:
what if do these thing in postversion?

Reply:
Good suggestion. I have updated.